src/tests/unit_tests/util/ld_util_test.py

```diff
@@ -17,47 +23,88 @@
 DUMMY_URI_NISV_ORGANISATION = "https://www.beeldengeluid.nl/"  # see setting_example.py
 
 
-# noinspection PyTypeChecker
 @pytest.mark.parametrize(
     "resource_level, resource_id, beng_data_domain, resource_uri",
     [
-        (
+        *[
+            (
                 level,  # program, series, season, scene, dataset, datacatalog, datadownload
                 DUMMY_RESOURCE_ID,
                 DUMMY_BENG_DATA_DOMAIN,
-                f"{DUMMY_BENG_DATA_DOMAIN}id/{level.value}/{DUMMY_RESOURCE_ID}"
-        ) for level in ResourceURILevel
-    ] + [
-        (ResourceURILevel.PROGRAM, DUMMY_RESOURCE_ID, "BROKEN_BENG_DATA_DOMAIN", None),  # invalid beng_data_domain
-        ("dataset", DUMMY_RESOURCE_ID, DUMMY_RESOURCE_ID, None),  # invalid level param (no str allowed)
-        ("program", DUMMY_RESOURCE_ID, DUMMY_RESOURCE_ID, None)  # another invalid level param
+                f"{DUMMY_BENG_DATA_DOMAIN}id/{level.value}/{DUMMY_RESOURCE_ID}",
+            )
+            for level in ResourceURILevel
+        ],
+        (
+            ResourceURILevel.PROGRAM,
+            DUMMY_RESOURCE_ID,
+            "BROKEN_BENG_DATA_DOMAIN",
+            None,
+        ),  # invalid beng_data_domain
+        (
+            "dataset",
+            DUMMY_RESOURCE_ID,
+            DUMMY_RESOURCE_ID,
+            None,
+        ),  # invalid level param (no str allowed)
+        (
+            "program",
+            DUMMY_RESOURCE_ID,
+            DUMMY_RESOURCE_ID,
+            None,
+        ),  # another invalid level param
     ],
 )
-def test_generate_lod_resource_uri(resource_level, resource_id, beng_data_domain, resource_uri):
+def test_generate_lod_resource_uri(
+    resource_level, resource_id, beng_data_domain, resource_uri
+):
     try:
-        assert generate_lod_resource_uri(resource_level, resource_id, beng_data_domain) == resource_uri
+        assert (
+            generate_lod_resource_uri(resource_level, resource_id, beng_data_domain)
+            == resource_uri
+        )
     finally:
         unstub()
 
 
 @pytest.mark.parametrize(
     "resource_url, sparql_endpoint, nisv_organisation_uri, raise_connection_error, success",
     [
-        (DUMMY_RESOURCE_URI, DUMMY_SPARQL_ENDPOINT, DUMMY_URI_NISV_ORGANISATION, False, True),
-        (DUMMY_RESOURCE_URI, DUMMY_SPARQL_ENDPOINT, DUMMY_URI_NISV_ORGANISATION, True, False),
+        (
```

Comment:
is it necessary to put this parameter set on multiple lines? And why are not all lines split? What is our preference... it does make the tests a bit less compact and maybe less uncluttered